### PR TITLE
♻️  Sidesheet story to Typescript

### PR DESCRIPTION
**Pull request by pomfrida**
Part of #735

### CODE REVIEW

**Comment by pomfrida** [comment]
Since we now have controls in story, I think an example page for each variant is a bit overkill - so I removed a lot of code.

File: apps/storybook-react/stories/SideSheet.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+export default {
+  title: 'Components/SideSheet',
+  component: SideSheet,
+  argTypes: {
+    variant: {
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium', 'large', 'xlarge'],
```
Comment (wenche) [inline apps/storybook-react/stories/SideSheet.stories.tsx:22]: Can you avoid duplicating this here? I _think_ @vnys  did something here for the, hmm can't remember exactly which component, table maybe?

**Review by wenche — CHANGES_REQUESTED** [review]
Just a small comment on tight coupling

File: apps/storybook-react/stories/SideSheet.stories.tsx (hunk 1 above)
Comment (pomfrida) [reply apps/storybook-react/stories/SideSheet.stories.tsx]: Yup, this code was pretty much useless, just removed it 😄

**Review by wenche — APPROVED** [review]
LGTM! 👍

**Pull request merged**